templates/rag-chroma-multi-modal/rag_chroma_multi_modal/ingest.py

```diff
@@ -0,0 +1,50 @@
+import os
+from pathlib import Path
+
+import pypdfium2 as pdfium
+from langchain.vectorstores import Chroma
+from langchain_experimental.open_clip import OpenCLIPEmbeddings
+
+
+def get_images_from_pdf(pdf_path, img_dump_path):
+    """
+    Extract images from each page of a PDF document and save as JPEG files.
+
+    :param pdf_path: A string representing the path to the PDF file.
+    :param img_dump_path: A string representing the path to dummp images.
+    """
+    pdf = pdfium.PdfDocument(pdf_path)
+    n_pages = len(pdf)
+    for page_number in range(n_pages):
+        page = pdf.get_page(page_number)
+        bitmap = page.render(scale=1, rotation=0, crop=(0, 0, 0, 0))
+        pil_image = bitmap.to_pil()
+        pil_image.save(f"{img_dump_path}/img_{page_number + 1}.jpg", format="JPEG")
+
+
+doc_path = Path(__file__).parent / "docs/DDOG_Q3_earnings_deck.pdf"
+img_dump_path = Path(__file__).parent / "docs/"
```

Comment:
would it be good to dump images in an inner directory that we can gitignore? Or a different directory?